src/modules/Elsa.EntityFrameworkCore/Extensions/ExpressionExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace Elsa.EntityFrameworkCore.Extensions;
+
+public static class ExpressionExtensions
+{
+    public static PropertyInfo GetPropertyInfo<TType, TReturn>(this Expression<Func<TType, TReturn>> propertyExpression)
```

Comment:
I think you should be able to reuse the `PropertyAccessorExtensions` found in `Elsa.Workflows.Core`.

Reply:
Done.